jansi/src/main/java/org/fusesource/jansi/AnsiConsole.java

```diff
@@ -129,7 +129,7 @@ public void close() throws IOException {
         };
     }
 
-    private static boolean isCygwin() {
+    private static boolean isXterm() {
         String term = System.getenv("TERM");
         return term != null && term.equals("xterm");
```

Comment:
notice: this could be changed to simply "xterm".equals(System.getenv("TERM"))
while at it...